docs/madr/decisions/046-production-installation-values.md

```diff
@@ -0,0 +1,146 @@
+
+### Stability
+
+Improves stability by:
+- prevent pod disruptions by enabling the pod disruption budgets for control plane, ingress and egress components 
+- ensuring a minimum number of 2 replicas for the control plane, ingress and egress components, and replicas are preferred to be scheduled onto different nodes
+- ensuring resource limits identical to requests so that pod resources will not be squeezed by other applications on the cluster
+- setting metrics for triggering HPA to 50% CPU and memory usage to make sure pods are not over-utilized when other instances are crashing
+
+### Performance
+
+Improves performance by:
+- enabling HPAs for the control plane, ingress and egress components
+- setting metrics for triggering HPA to 50% CPU and memory usage to make sure pods are performing well
+
+### Proposed production installation values
+
+The values here will override the default `values.yaml` file, so refer to [the existing file](https://github.com/kumahq/kuma/blob/master/deployments/charts/kuma/values.yaml) if needed. 
+
+```yaml
+
+controlPlane:
+  # -- Used in `zone` mode with `kdsGlobalAddress` is not empty
+  kdsGlobalAuth:
+    type: "token"
+    token: ""
+    
+  # this is a new key to restrict access to the Kuma control plane api server by cores
+  apiServer:
+    coresAllowedDomains:
+      - "https://localhost:5681"
+      - "http://localhost:5681"
+
+  replicas: 2
+
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 1024Mi
+    limits:
+      cpu: 1000m
+      memory: 1024Mi
+
+  autoscaling:
+    enabled: true
+
+    # -- For clusters that don't support autoscaling/v2, autoscaling/v1 is used
+    targetCPUUtilizationPercentage: 50
+    # -- For clusters that do support autoscaling/v2, use metrics
+    metrics:
+      - type: Resource
+        resource:
+          name: cpu
+          target:
+            type: Utilization
+            averageUtilization: 50
+      - type: Resource
+        resource:
+          name: memory
+          target:
+            type: Utilization
+            averageUtilization: 50
+
+
+  podDisruptionBudget:
+    enabled: true
+
+cni:
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+    limits:
+      cpu: 100m
+      memory: 100Mi
+
+ingress:
+  replicas: 2
+
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 1024Mi
+    limits:
+      cpu: 1000m
+      memory: 1024Mi
+
+  autoscaling:
+    enabled: true
+    # -- For clusters that don't support autoscaling/v2, autoscaling/v1 is used
+    targetCPUUtilizationPercentage: 50
+    # -- For clusters that do support autoscaling/v2, use metrics
+    metrics:
+      - type: Resource
+        resource:
+          name: cpu
+          target:
+            type: Utilization
+            averageUtilization: 50
+      - type: Resource
+        resource:
+          name: memory
+          target:
+            type: Utilization
+            averageUtilization: 50
```

Comment:
Is this a little low? Should it be more like 80? What do other popular charts do?

Reply:
Let me check other implementations.